### ISSUE TITLE
Accessing statistics tab breaks program

### ISSUE THREAD

**Issue by HighlyTrainedNeuralNetwork** [cderickson/MTGO-Tracker#27]
After recently importing some gamelogs if I try to access the "Statistics" tab I only see four empty quadrants. 
![image](https://github.com/cderickson/MTGO-Tracker/assets/56235713/89062210-bc87-4378-92f3-fda0cbc7d135)
Additionally if I try to close this and return to the main menu, the entire program closes. This is occurring on both 0.13 and 0.15 however 0.15 does not seem to have a "DEBUG.txt" file so I am attaching a zip of the one from 0.13.
[DEBUG.zip](https://github.com/cderickson/MTGO-Tracker/files/12740416/DEBUG.zip)

**Comment by cderickson** [comment]
This is probably a similar issue to the one you submitted before (https://github.com/cderickson/MTGO-Tracker/issues/24) where it's dividing by zero somewhere. Can you upload a zip of your /save folder and I can debug on my machine when I get home from work? Thanks

**Comment by HighlyTrainedNeuralNetwork** [comment]
Makes sense. Yeah here it is:
[save.zip](https://github.com/cderickson/MTGO-Tracker/files/12742110/save.zip)

**Comment by cderickson** [comment]
This issue was caused due to an error processing match#: **202309181348_Snusnumrick_aManatease**

Without the actual GameLog file, I can't tell why this match couldn't be processed, but future MTGO-Tracker versions will skip matches like this when there is an error in processing so the program won't break.

For right now, you can select this match and click **Remove Record(s) -> Remove and Ignore**

Let me know if you have any other issues

**Comment by cderickson** [comment]
Are you able to attach a zip of your GameLog files (or just the single file, it would have a file creation time of **2023-09-18 1:48pm**)

I added a change in version 15 that should have skipped this file while importing but it obviously didn't work and I'd like to fix it for the next release.

Sorry for asking so much of you but I'd like to get this fixed lol. Appreciate your help

**Comment by HighlyTrainedNeuralNetwork** [comment]
Yeah sure thing! Nothing to be sorry for, you've put a ton of work into an awesome tool and I appreciate the support. Here it is:
[Match_GameLog_f0d2e4f4-7a07-432b-9b78-e2692f7b40cc.zip](https://github.com/cderickson/MTGO-Tracker/files/12755079/Match_GameLog_f0d2e4f4-7a07-432b-9b78-e2692f7b40cc.zip)

**Comment by cderickson** [comment]
It looks like you lost connection and reconnected, which resulted in a partial log file (there was only data for Game 1).

I added a fix for this in the most recent update (v15) and tested this using your GameLog successfully.

I'm guessing you imported using an older version (v13?) and then transferred your save data to a newer version. In any case, I've added a startup check that will automatically delete "bad" data like this so users can take save data from older versions safely.

The simplest fix for you would be to **Remove Record(s) -> Remove and Ignore** for that particular match.

Thanks for your help investigating this issue!
